lib/pf/activation.pm

```diff
@@ -608,9 +606,18 @@ sub sms_activation_create_send {
 
     my $activation_code = create(\%args);
     if (defined($activation_code)) {
-      unless (send_sms($activation_code, %info)) {
-        ($success, $err) = ($FALSE, $GUEST::ERROR_CONFIRMATION_SMS);
-      }
+        if ( $authentication_source->can_send_sms ) {
+            unless ($authentication_source->sendSMS($activation_code)) {
+                ($success, $err) = ($FALSE, $GUEST::ERRORS{$GUEST::ERROR_CONFIRMATION_SMS});
+                invalidate_codes($args{'mac'}, $args{'pid'}, $args{'pending'});
+            }
+        }
+        else {
+            unless (send_sms($activation_code)) {
```

Comment:
send_sms($activation_code,%info), since there is no more %info param, don't we have to remove it in the sub send_sms function ?